src/main/java/bit/javaoop/Company.java

```diff
@@ -0,0 +1,46 @@
+package bit.javaoop;
+
+public class Company {
+    private String companyName;
+    private CEO ceo;
+
+
+    public void setCompanyName(String companyName) {
+        this.companyName = companyName;
+    }
+
+    public CEO hireCEO()
+    {
+        CEO ceo1 = CEO.getInstance();
+        ceo=ceo1;
+        ceo.setName("Krzychu"); //setting name e.g from keyboard
```

Comment:
It'd be much better to pass a `CEO` as a constructor parameter or set it somewhere outside. Now your Company is tightly couplet to this one `CEO` but it should be possible to change company's CEO.